# Bibliography

\[1\] E. M. ä, M. Tolonen, J. Marjanen, et al. “Interdisciplinary
collaboration in studying newspaper materiality.” In: *Proc. Twin Talks
workshop. Digital Humanities in the Nordic Countries*. Ed. by S. Krauwer
and D. FiÅ¡er. Vol. 2365. CEUR Workshop Proceedings. Copenhagen, mars.
2019, pp. 55-66. &lt;URL:
<http://ceur-ws.org/Vol-2365/07-TwinTalks-DHN2019_paper_7.pdf>&gt;.

\[2\] A. Aatsinki, A. Keskitalo, V. Laitinen, et al. “Maternal prenatal
psychological distress and hair cortisol levels associate with infant
fecal microbiota composition at 2.5 months of age.” In:
*Psychoneuroendocrinology* 119 (2020), p. 104754. DOI:
10.1016/j.psyneuen.2020.104754.

\[3\] A. Aatsinki, L. Lahti, H. Uusitupa, et al. “Gut microbiota
composition is associated with temperament traits in infants.” In:
*Brain Behavior and Immunity* 80 (mai. 2019), pp. 849-858. DOI:
10.1016/j.bbi.2019.05.035.

\[4\] A. Aatsinki, H. Uusitupa, E. Munukka, et al. “Gut Microbiota
Composition in Mid-Pregnancy Is Associated with Gestational Weight Gain
but Not Prepregnancy Body Mass Index.” In: *Journal of Womens Health*
(mai. 2018). Epub ahead of print. DOI: 10.1089/jwh.2017.6488.

\[5\] J. Aleksic, A. Alexa, T. K. Attwood, et al. “The Open Science Peer
Review Oath.” In: *F1000Research* 3 (janv.. 2015). AllBio: Open Science
& Reproducibility Best Practice Workshop, p. 271. DOI:
10.12688/f1000research.5686.2.

\[6\] J. Alneberg, B. S. Bjarnason, I. de Bruijn, et al. “Binning
metagenomic contigs by coverage and composition.” In: *Nature Methods*
11.1144-1146 (2014). CONCOCT algorithm. DOI: 10.1038/nmeth.3103.

\[7\] B. M. S. Arani, M. Mahmoudi, L. Lahti, et al. “Stability
estimation of autoregulated genes under Michaelis-Menten type kinetics.”
In: *Physical Review E* 97.6 (juin. 2018), p. 062407. ISSN: 2470-0045.
DOI: 10.1103/PhysRevE.97.062407. &lt;URL:
<https://link.aps.org/doi/10.1103/PhysRevE.97.062407>&gt;.

\[8\] S. Atashgahi, R. Aydin, M. R. Dimitrov, et al. “Impact of a
wastewater treatment plant on microbial community composition and
function in a hyporheic zone of a eutrophic river.” In: *Scientific
Reports* 5.17284 (nov.. 2015). DOI: 10.1038/srep17284.

\[9\] A. BjÃ¶rk, J. Paavolainen, T. Ropponen, et al. *Opening academic
publishing: development and application of systematic evaluation
criteria*. Tech. rep. Report on the openness of major scientific
publishers commissioned by Finnish Ministry of Education and Culture.
unknown, janv.. 2018. &lt;URL:
<http://urn.fi/URN:NBN:fi-fe201802123334>&gt;.

\[10\] I. Borze, M. Guled, S. Musse, et al. “MicroRNA microarrays on
archive bone marrow core biopsies of leukemias - method validation.” In:
*Leukemia Research* 35 (2 fÃ©vr.. 2011), pp. 188-195. DOI:
10.1016/j.leukres.2010.08.005.

\[11\] E. Buelow, T. d. B. GonzÃƒÂ¡lez, S. Fuentes, et al. “Comparative
gut microbiota and resistome profiling of intensive care patients
receiving selective digestive tract decontamination and healthy
subjects.” In: *Microbiome* 5.1 (août. 2017), p. 88. DOI:
10.1186/s40168-017-0309-z.

\[12\] M. Eetu, K. Lagus, L. Lahti, et al. “Wrangling with Non-Standard
Data.” In: *Proc. Digital Humanities in the Nordic Countries*. Ed. by S.
Reinsone, I. SkadiÅ†a, A. BaklÄne and J. Daugavietis. Vol. 2612. CEUR
Workshop Proceedings. mars. 2020, pp. 81-96.

\[13\] L. L. Elo, L. Lahti, H. Skottman, et al. “Integrating probe-level
expression changes across generations of Affymetrix arrays.” In:
*Nucleic Acids Research* 33.22 (déc.. 2005), p. e193. DOI:
10.1093/nar/gni193.

\[14\] A. Faisal, R. Louhimo, L. Lahti, et al. “Biomarker discovery via
dependency analysis of multi-view functional genomics data.” In: *NIPS
2011 workshop From Statistical Genetics to Predictive Models in
Personalized Medicine*. 2011.

\[15\] K. Faust, F. Bauchinger, B. Laroche, et al. “Signatures of
ecological processes in microbial community time series.” In:
*Microbiome* 6.120 (juin. 2018). DOI: 10.1186/s40168-018-0496-2.

\[16\] K. Faust, L. Lahti, D. Gonze, et al. “Metagenomics meets time
series analysis: unraveling microbial community dynamics.” In: *Current
Opinion in Microbiology* 25 (juin. 2015), pp. 56-66. DOI:
10.1016/j.mib.2015.04.004.

\[17\] D. Gonze, K. Z. Coyte, L. Lahti, et al. “Microbial communities as
dynamical systems.” In: *Current Opinion in Microbiology* 44 (juil..
2018), pp. 41-49. DOI: 10.1016/j.mib.2018.07.004.

\[18\] D. Gonze, L. Lahti, J. Raes, et al. “Multi-stability and the
origin of microbial community types.” In: *ISME Journal* 11 (mai. 2017),
pp. 2159-2166. DOI: 10.1038/ismej.2017.60.

\[19\] M. Guled, L. Lahti, P. M. Lindholm, et al. “CDKN2A, NF2 and JUN
Are Dysregulated Among Other Genes by miRNAs in Malignant Mesothelioma -
a miRNA Microarray Analysis.” In: *Genes, Chromosomes and Cancer* 48.7
(juil.. 2009), pp. 615-623. DOI: 10.1002/gcc.20669.

\[20\] K. Harris, T. L. Parsons, U. Z. Ijaz, et al. “Linking statistical
and ecological theory: Hubbell’s unified neutral theory of biodiversity
as a hierarchical Dirichlet process.” In: *Proceedings of the IEEE*
105.3 (mars. 2017), pp. 516-529. DOI: 10.1109/JPROC.2015.2428213.

\[21\] M. A. Heiskanen, S. M. Honkala, R. A. Ojala, et al. “Systemic
cross-talk between brain, gut, and peripheral tissues in glucose
homeostasis: effects of exercise training (CROSSYS): Exercise training
intervention in monozygotic twins discordant for body weight.” In: *BMC
Sports Science, Medicine and Rehabilitation* 13 (2021), p. 16. DOI:
10.1186/s13102-021-00241-z.

\[22\] M. J. Hill, A. Kanner, J. Marjanen, et al. “Spheres of
â€œpublicâ€ in eighteenth-century Britain.” In: *Proceedings of the
Digital Humanities in the Nordics Conference*. Helsinki, Finland, mars.
2018.

\[23\] M. J. Hill, V. Vaara, T. SÃ¤ily, et al. “Reconstructing
intellectual networks: from the ESTC’s bibliographic metadata to
historical material.” In: *Proc. Digital Humanities in the Nordic
Countries*. Ed. by C. Navarretta, M. Agirrezabal and B. Maegaard. Vol.
2364. CEUR Workshop Proceedings. Best paper award.. Copenhagen, mars.
2019, pp. 201-219. &lt;URL:
<https://cst.dk/DHN2019Pro/papers/DHN2019_hill.pdf>&gt;.

\[24\] J. Hintikka, S. Lensu, E. MÃ¤kinen, et al. “Xylo-Oligosaccharides
in Prevention of Hepatic Steatosis and Adipose Tissue Inflammation:
Associating Taxonomic and Metabolomic Patterns in Fecal Microbiomes with
Biclustering.” In: *International Journal of Environmental Research and
Public Health* 18.8 (avr.. 2021), p. 4049. DOI: 10.3390/ijerph18084049 .

\[25\] O. Huovilainen and L. Lahti. *Pairwise integration of functional
genomics data*. software. R/BioConductor: pint. avr.. 2010. &lt;URL:
<http://bioconductor.org/packages/devel/bioc/html/pint.html>&gt;.

\[26\] A. Ijaz, H. Roivainen, and L. Lahti. “Analytical Edition
Detection In Bibliographic Metadata.” In: *Proceedings of the Digital
Humanities (DH2019)*. In press.. juil.. 2019.

\[27\] A. Ijaz, M. Tolonen, and L. Lahti. “Analytical determination of
editions from bibliographic metadata.” In: *Proceedings of the Research
Data in the Humanities Conference*. Oulu, août. 2019.

\[28\] J. Jalanka-Tuovinen, A. Salonen, J. N. ä, et al. “Intestinal
microbiota in healthy adults: Temporal analysis reveals individual and
common core and relation to intestinal symptoms.” In: *PLoS One* 6.7
(juil.. 2011), p. e23035. &lt;URL: 10.1371/journal.pone.0023035&gt;.

\[29\] K. T. K, F. J. F, M. R. M, et al. “Microbiome yarns: The Global
Phenotype-Genotype Survey. Episode III: importance of microbiota
diversification for microbiome function and biome health.” In:
*Microbial Biotechnology* (avr.. 2019). DOI: 10.1111/1751-7915.13406.

\[30\] S. Kaski, J. N. ä, J. Sinkkonen, et al. “Associative clustering
for exploring dependencies between functional genomics data sets.” In:
*IEEE/ACM Transactions on Computational Biology and Bioinformatics* 2.3
(sept.. 2005). Special Issue on Machine Learning for Bioinformatics -
Part 2, pp. 203-216. DOI: 10.1109/TCBB.2005.32.

\[31\] A. Keskitalo and others. “Gut microbiota diversity but not
composition is related to saliva cortisol stress response at the age of
2.5 months.” In: *Stress* (2021). In press.

\[32\] M. Khalighi, L. Eftekhari, S. Hosseinpour, et al. “Three-species
Lotka-Volterra model with respect to Caputo and Caputo-Fabrizio
fractional operators.” In: *Symmetry* 13 (3 janv.. 2020), p. 368. DOI:
10.3390/sym13030368.

\[33\] J. Koffert, L. Lahti, L. Nylund, et al. “Partial restoration of
normal intestinal microbiota in morbidly obese women six months after
bariatric surgery.” In: *PeerJ Ã¢â‚¬â€œ Life & Environment* 8 (2020).
Shared first authorship, p. e10442. DOI: 10.7717/peerj.10442.

\[34\] L. Lahti. *A brief overview on the BioPAX and SBML standards*.
Finnish. arXiv. Overview of machine-readable representations of
biological processes.. 2007. &lt;URL:
<https://arxiv.org/abs/1109.4919>&gt;.

\[35\] L. Lahti. “Avoimuus voi vahvistaa pÃ¤Ã¤tÃ¶ksenteon tieteellistÃ¤
pohjaa.” In: *University of Helsinki, ThinkOpen blog* (août. 2020).
&lt;URL:
<https://blogs.helsinki.fi/thinkopen/lahdekoodin-avoimuus-paatoksenteossa/>&gt;.

\[36\] L. Lahti. “Comparative functional genome analysis using
associative clustering.” Finnish. MA Thesis. (in Finnish). Helsinki
University of Technology, déc.. 2003. &lt;URL:
<https://github.com/antagomir/thesis/tree/master/msc03>&gt;.

\[37\] L. Lahti. *Concerning Blackburn’s quasi-realism in securing moral
discussion*. Finnish. B.Sc. thesis. University of Helsinki, Faculty of
Political Sciences. (In Finnish). juil.. 2009.

\[38\] L. Lahti. “Intestinal microbiota, individuality and health. In:
Computational Methods Aiding Early-Stage Drug Design (Dagstuhl Seminar
13212).” In: *Dagstuhl Reports* 3.5 (2013). Ed. by A. Bender, H.
GÃƒÂ¶hlmann, S. Hochreiter and Z. Shkedy, pp. 78-94. ISSN: 2192-5283.
DOI: 10.4230/DagRep.3.5.78. &lt;URL:
<http://drops.dagstuhl.de/opus/volltexte/2013/4179>&gt;.

\[39\] L. Lahti. *Itsesimilaaristen joukkojen Hausdorffin dimension
mÃ¤Ã¤rittÃ¤minen*. Finnish. Special assignment in Mathematics (in
Finnish). 2002. &lt;URL:
<https://github.com/antagomir/thesis/tree/master/hausdorff02>&gt;.

\[40\] L. Lahti. *Julian joukko iteraatioden valuma-altaan reunana*.
Finnish. Special assignment in Mathematics (in Finnish). 2001. &lt;URL:
<https://github.com/antagomir/thesis/tree/master/julia01>&gt;.

\[41\] L. Lahti. “LÃ¤hdekoodin sisÃ¤ltÃ¤mien yksityiskohtien avoimuus on
keskeistÃ¤ pÃ¤Ã¤tÃ¶ksenteon lÃ¤pinÃ¤kyvyydelle.” In: *HS Mielipide*
(juin. 2020). &lt;URL:
<https://www.hs.fi/mielipide/art-2000006545770.html>&gt;.

\[42\] L. Lahti. “Open Data Science.” In: *Advances in Intelligent Data
Analysis XVII. Lecture Notes in Computer Science 11191.* Vol. 11191.
Conference proceedings.. India: Springer, oct.. 2018.

\[43\] L. Lahti. *opencomp blog*. &lt;URL:
<http://antagomir.wordpress.com/>&gt;.

\[44\] L. Lahti. “PÃ¤Ã¤tÃ¶s epidemialaskelmien salaamisesta vastoin
valtioneuvoston avoimuuslinjausta.” In: *Open Knowledge Finland Blog*
(juin. 2020). &lt;URL:
<https://www.okf.fi/fi/2020/06/15/thln-paatos-epidemialaskelmien-salaamisesta-vastoin-valtioneuvoston-avoimuuslinjausta/>&gt;.

\[45\] L. Lahti. “Probabilistic analysis of the human transcriptome with
side information.” Finnish. PhD thesis. LaTeX sources and the pdf
version are openly available:
<https://github.com/antagomir/thesis/tree/master/phd10>; Press release:
<https://github.com/antagomir/thesis/blob/master/phd10/Vaitostiedote-LeoLahti-20101207.pdf>.
Espoo: Aalto University School of Science and Technology, Faculty of
Information and Natural Sciences, Department of Information and Computer
Science, déc.. 2010. &lt;URL:
<http://lib.tkk.fi/Diss/2010/isbn9789526033686/>&gt;.

\[46\] L. Lahti. *RPA: probe reliability and differential gene
expression analysis*. software. R/Bioconductor: RPA. oct.. 2009. DOI:
10.18129/B9.BIOC.RPA. &lt;URL:
<http://bioconductor.org/packages/release/bioc/html/RPA.html>&gt;.

\[47\] L. Lahti, L. L. Elo, T. Aittokallio, et al. “Probabilistic
Analysis of Probe Reliability in Differential Gene Expression Studies
with Short Oligonucleotide Arrays.” In: *IEEE/ACM Transactions on
Computational Biology and Bioinformatics* 8.1 (janv.. 2011), pp.
217-225. DOI: 10.1109/TCBB.2009.38.

\[48\] L. Lahti, A. Gusmao, and O. Huovilainen. *NetResponse (functional
network analysis)*. software. R/Bioconductor: netresponse. oct.. 2010.
DOI: 10.18129/B9.BIOC.NETRESPONSE. &lt;URL:
<http://www.bioconductor.org/help/bioc-views/devel/bioc/html/netresponse.html>&gt;.

\[49\] L. Lahti, J. Huovari, M. Kainu, et al. “Retrieval and Analysis of
Eurostat Open Data with the eurostat Package.” In: *The R Journal* 9.1
(2017), pp. 385-392. &lt;URL:
<https://journal.r-project.org/archive/2017/RJ-2017-019/index.html>&gt;.

\[50\] L. Lahti, N. IlomÃ¤ki, and M. Tolonen. “A quantitative study of
history in the English short-title catalogue (ESTC) 1470-1800.” In:
*LIBER Quarterly* 25.2 (déc.. 2015), pp. 87-116. ISSN: 2213-056X. DOI:
10.18352/lq.10112.

\[51\] L. Lahti and S. Kaski. *Probabilistic dependency models for data
integration in functional genomics*. Machine Learning for Systems
Biology workshop, ISMB, Vienna, Austria. juil.. 2011.

\[52\] L. Lahti, J. E. Knuuttila, and S. Kaski. “Global modeling of
transcriptional responses in interaction networks.” In: *Bioinformatics*
26 (21 nov.. 2010). R/Matlab implementations available at
<http://netpro.r-forge.r-project.org/>, pp. 2713-2720. DOI:
10.1093/bioinformatics/btq500. &lt;URL:
<http://bioinformatics.oxfordjournals.org/content/early/2010/09/02/bioinformatics.btq500.abstract.html?ijkey=N1zzjtUBRng3wCM&keytype=ref>&gt;.

\[53\] L. Lahti, J. LehtomÃ¤ki, M. Kainu, et al. *rOpenGov blog*. Open
government data analytics blog. &lt;URL:
<http://ropengov.github.io/>&gt;.

\[54\] L. Lahti, J. LehtomÃ¤ki, J. Parkkinen, et al. *Louhos blog*. Open
government data analytics blog (in Finnish). &lt;URL:
<http://louhos.github.io/>&gt;.

\[55\] L. Lahti, E. MÃ¤kelÃ¤, and M. Tolonen. “Quantifying bias and
uncertainty in historical data collections with probabilistic
programming.” In: *Proc. CEUR Workshop Proceedings on Computational
Humanities Research*. nov.. 2020, pp. 80-89. &lt;URL:
<http://ceur-ws.org/Vol-2723/short46.pdf>&gt;.

\[56\] L. Lahti, J. Marjanen, H. Roivainen, et al. “Bibliographic Data
Science and the History of the Book (c. 1500â€“1800).” In: *Cataloging &
Classification Quarterly* 57.1 (janv.. 2019). Special issue, pp. 5-23.
DOI: 10.1080/01639374.2018.1543747.

\[57\] L. Lahti, S. Myllykangas, S. Knuutila, et al. “Dependency
detection with similarity constraints.” In: *Proc. MLSP’09 IEEE
International Workshop on Machine Learning for Signal Processing XIX*.
Ed. by J. C. Adali T Chanussot J and L. J. R/Bioconductor: pint.
Piscataway, NJ, 2009, pp. 198-203. &lt;URL:
<http://arxiv.org/abs/1101.5919>&gt;.

\[58\] L. Lahti, J. SalojÃ¤rvi, A. Salonen, et al. “Tipping elements in
the human intestinal ecosystem.” In: *Nature Communications* 5 (juil..
2014), p. 4344. DOI: 10.1038/ncomms5344.

\[59\] L. Lahti, A. Salonen, R. A. Kekkonen, et al. “Associations
between the human intestinal microbiota, Lactobacillus rhamnosus GG and
serum lipids indicated by integrated analysis of high-throughput
profiling data.” In: *PeerJ* 1 (2013). Highly accessed, featured in
PeerJ journal blog and Top-10 Microbiology collection, p. e32. DOI:
10.7717/peerj.32.

\[60\] L. Lahti, M. SchÃ¤fer, H. Klein, et al. “Cancer gene
prioritization by integrative analysis of mRNA expression and DNA copy
number data: a comparative review.” In: *Briefings in Bioinformatics*
14.1 (juin. 2013), pp. 27-35. DOI: 10.1093/bib/bbs005.

\[61\] L. Lahti, M. SchÃ¤fer, H. Klein, et al. *intcomp: a benchmarking
pipeline for integrative cancer gene detection algorithms*. software.
R-forge. févr.. 2011. &lt;URL:
<http://intcomp.r-forge.r-project.org/>&gt;.

\[62\] L. Lahti, F. da Silva, M. P. Laine, et al. “Alchemy & algorithms:
perspectives on the philosophy and history of open science.” In: *RIO
Journal* 3 (mai. 2017). Review of the international conference,
including electronic material (video interviews, lectures and podcasts,
p. e13593. DOI: 10.3897/rio.3.e13593.

\[63\] L. Lahti and T. Toikkanen. “TietopyyntÃ¶ epidemialaskelmien
lÃ¤hdekoodeista.” In: *Open Knowledge Finland Blog* (mai. 2020).
&lt;URL:
<https://www.okf.fi/fi/2020/05/13/tietopyynto-thln-epidemialaskelmien-lahdekoodeista>&gt;.

\[64\] L. Lahti, A. Torrente, L. L. Elo, et al. “A fully scalable online
pre-processing algorithm for short oligonucleotide microarray atlases.”
In: *Nucleic Acids Research* 41.10 (2013). Implementation available
through Bioconductor RPA package, p. e110. DOI: 10.1093/nar/gkt229.

\[65\] L. Lahti, A. Tripathi, and O. Huovilainen. *Dependency modeling
toolkit*. software. CRAN: dmt. juin. 2010. &lt;URL:
<http://dmt.r-forge.r-project.org/>&gt;.

\[66\] L. Lahti, V. Vaara, J. Marjanen, et al. “Best practices in
bibliographic data science.” In: *Proceedings of the Research Data in
the Humanities Conference*. Oulu, août. 2019.

\[67\] L. Lahti, T. Wallgren, and M. Kulmala. “Laskentamallit eivÃ¤t
lÃ¤htÃ¶kohtaisesti ole salassa pidettÃ¤vÃ¤Ã¤ tietoa.” In: *HS Mielipide*
(mai. 2020). &lt;URL:
<https://www.hs.fi/mielipide/art-2000006494641.html>&gt;.

\[68\] H. Laine, L. Lahti, A. Lehto, et al. “Beyond Open Access - The
Changing Culture of Producing and Disseminating Scientific Knowledge.”
In: *Proceedings of the 19th International Academic Mindtrek Conference
in Tampere, Finland, September 22-24*. AcademicMindTrek’15: Proceedings
of the 19th International Academic Mindtrek Conference. Tampere,
Finland: ACM, 2015. ISBN: 978-1-4503-3948-3. &lt;URL:
<http://dl.acm.org/citation.cfm?id=2818187>&gt;.

\[69\] V. Laitinen and L. Lahti. “A hierarchical Ornstein-Uhlenbeck
model for stochastic time series analysis.” In: *Advances in Intelligent
Data Analysis XVII. Lecture Notes in Computer Science 11191.* Conference
proceedings.. India: Springer, oct.. 2018. &lt;URL:
<https://github.com/velait/OUP>&gt;.

\[70\] S. Lensu, R. Pariyani, E. MÃ¤kinen, et al. “Prebiotic
Xylo-oligosaccharides Targeting Faecalibacterium prausnitzii Prevent
High Fat Diet-induced Hepatic Steatosis in Rats.” In: *Nutrients* 12.11
(oct.. 2020), p. 3225. DOI: 10.3390/nu12113225.

\[71\] J. L. Leo Lahti Juuso Parkkinen and M. Kainu. *rOpenGov: open
source ecosystem for computational social sciences and digital
humanities*. software. ICML/MLOSS workshop (Int’l Conf. on Machine
Learning - Open Source Software workshop).. déc.. 2013. &lt;URL:
<http://ropengov.github.io>&gt;.

\[72\] Y. Liu, G. Meric, A. S. Havulinna, et al. “Early prediction of
liver disease using conventional risk factors and gut
microbiome-augmented gradient boosting.” In: *medRxiv* (2020). DOI:
10.1101/2020.06.24.20138933. eprint:
<https://www.medrxiv.org/content/early/2020/06/25/2020.06.24.20138933.full.pdf>.
&lt;URL:
<https://www.medrxiv.org/content/early/2020/06/25/2020.06.24.20138933>&gt;.

\[73\] R. Louhimo, V. AittomÃ¤ki, A. Faisal, et al. “Systematic Use of
Computational Methods Allows Stratifying Treatment Responders in
Glioblastoma Multiforme.” In: *Systems Biomedicine (special issue)* 1 (2
avr.. 2014). Critical Assessment of Massive Data Analysis (CAMDA)
workshop, pp. 130-136. DOI: 10.4161/sysb.28904.

\[74\] Marcos-Zambrano, L. Judith, Karaduzovic-Hadziabdic, et al.
“Applications of Machine Learning in Human Microbiome Studies: A Review
on Feature Selection, Biomarker Identification, Disease Prediction and
Treatment.” In: *Frontiers in Microbiology* 12 (2021), p. 313. ISSN:
1664-302X. DOI: 10.3389/fmicb.2021.634511. &lt;URL:
<https://www.frontiersin.org/article/10.3389/fmicb.2021.634511>&gt;.

\[75\] J. Marjanen, V. Vaara, A. Kanner, et al. “A National Public
Sphere? Analyzing the Language, Location, and Form of Newspapers in
Finland, 1771â€“1917.” In: *Journal of European Periodical Studies* 4.1
(juin. 2019). Special issue: Digital Approaches Towards Serial
Publications. DOI: 10.21825/jeps.v4i1.10483. &lt;URL:
<https://ojs.ugent.be/jeps/article/view/10483>&gt;.

\[76\] I. Moreno-Indias, L. Lahti, M. Nedyalkova, et al. “Statistical
and machine learning techniques in human microbiome studies:
contemporary challenges and solutions.” In: *Frontiers in Microbiology*
12 (févr.. 2021), p. 277. DOI: 10.3389/fmicb.2021.635781.

\[77\] N. Mosakhani, M. Guled, L. Lahti, et al. “Unique microRNA profile
in Dupuytren’s contracture supports deregulation of beta-catenin
pathway.” In: *Modern Pathology* 23 (nov.. 2010). International
Dupuytren Award for best research paper in 2011, pp. 1544-1522. DOI:
10.1038/modpathol.2010.146.

\[78\] N. Mosakhani, L. Lahti, I. Borze, et al. “MicroRNA profiling
predicts survival in anti-EGFR treated chemorefractory metastatic
colorectal cancer patients with wild-type KRAS and BRAF.” In: *Cancer
Genetics* 205.11 (nov.. 2012), pp. 545-51. DOI:
10.1016/j.cancergen.2012.08.003.

\[79\] N. Mosakhani, V. Sarhadi, A. Usvasalo, et al. “MicroRNA profiling
in pediatric acute lymphoblastic leukemia: novel prognostic tools.” In:
*Leukemia & Lymphoma* 53.12 (déc.. 2012), pp. 2517-2520. DOI:
10.3109/10428194.2012.685731.

\[80\] T. Niini, L. Lahti, F. Michelacci, et al. “Array Comparative
Genomic Hybridization Reveals Frequent Alterations of G1/S Checkpoint
Genes in Undifferentiated Pleomorphic Sarcoma of Bone.” In: *Genes,
Chromosomes and Cancer* 50.5 (mai. 2011), pp. 291-306. DOI:
10.1002/gcc.20851.

\[81\] T. Niini, I. Scheinin, L. Lahti, et al. “Homozygous Deletions of
Cadherin Genes in Chondrosarcoma â€“ an Array CGH Study.” In: *Cancer
Genetics* 205.11 (nov.. 2012), pp. 588-93. DOI:
10.1016/j.cancergen.2012.09.007.

\[82\] L. Nylund, S. Hakkola, L. Lahti, et al. “Diet, perceived
intestinal well-being, fecal microbiota and short chain fatty acids in
oat-using subjects with celiac disease or gluten sensitivity.” In:
*Nutrients* 12 (9 août. 2020). Special issue, p. 2570. DOI:
10.3390/nu12092570.

\[83\] P. Nymark, M. Guled, I. Borze, et al. “Integrative Analysis of
microRNA, mRNA and aCGH Data Reveals Asbestos- and Histology-Related
Changes in Lung Cancer.” In: *Genes, Chromosomes and Cancer* 50.8
(aoÃ»t. 2011), pp. 585-597. DOI: 10.1002/gcc.20880.

\[84\] P. Nymark, P. M. Lindholm, M. V. Korpela, et al. “Gene Expression
Profiles in Asbestos-exposed Epithelial and Mesothelial Lung Cell
Lines.” In: *BMC Genomics* 8.62 (mars. 2007). DOI:
10.1186/1471-2164-8-62.

\[85\] S. O’Keefe, J. Li, L. Lahti, et al. “Fat, Fiber and Cancer Risk
in African, Americans and Rural Africans.” In: *Nature Communications* 6
(avr.. 2015). Top science article in Guardian on the release day, p.
6342. DOI: 10.1038/ncomms7342.

\[86\] E. P. in Open Education (Open Science Coordination. *National
policy and executive plan by the higher education and research community
for 2021-2025 - Policy component 1: Open access to educational
resources.* Tech. rep. Helsinki, Finland: unknown, 2020.

\[87\] J. Palmu, L. Lahti, and T. Niiranen. “Targeting gut microbiota to
treat hypertension: a systematic review.” In: *International Journal of
Environmental Research and Public Health* 18.3 (janv.. 2021), p. 1248.
DOI: 10.3390/ijerph18031248.

\[88\] J. Palmu and others. “Eicosanoid Inflammatory Mediators Are
Robustly Associated with Blood Pressure in the General Population.” In:
*Journal of American Heart Association* 9 (15 2020), p. e016641. DOI:
10.1161/JAHA.120.017598.

\[89\] J. Palmu, A. Salosensaari, A. S. Havulinna, et al. “Association
Between the Gut Microbiota and Blood Pressure in a Population Cohort of
6953 Individuals.” In: *Journal of the American Heart Association*
(juil.. 2020). DOI: 10.1161/JAHA.120.016641.

\[90\] J. Ritari, J. SalojÃ¤rvi, L. Lahti, et al. “Improved taxonomic
assignment of human intestinal 16S rRNA sequences by a dedicated
reference database.” In: *BMC Genomics* 16 (déc.. 2015), p. 1056. DOI:
10.1186/s12864-015-2265-y.

\[91\] M. O. Ruuskanen, F. Ã…berg, V. MÃ¤nnistÃ¶, et al. “Links between
gut microbiome composition and fatty liver disease in a large population
sample.” In: *Gut Microbes* 13 (1 mars. 2021). DOI:
10.1080/19490976.2021.1888673.

\[92\] M. Ruuskanen, G. Sommeria-Klein, A. Havulinna, et al. “Modeling
spatial patterns in host-associated microbial communities.” In:
*Environmental Microbiology* 23.5 (mai. 2021), pp. 2374-2388. DOI:
10.1111/1462-2920.15462.

\[93\] A. Salonen, L. Lahti, J. SalojÃ¤rvi, et al. “Impact of diet and
individual variation on intestinal microbiota composition and
fermentation products in obese men.” In: *ISME Journal* 8 (2014), pp.
2218-2230. DOI: 10.1038/ismej.2014.63.

\[94\] A. Salonen, J. SalojÃ¤rvi, L. Lahti, et al. “The adult intestinal
core microbiota is determined by analysis depth and health status.” In:
*Clinical Microbiology and Infection* 18.Suppl. 4 (2012), p. 16â€“20.
DOI: 10.1111/j.1469-0691.2012.03855.x.

\[95\] A. Salosensaari*, V. Laitinen*, A. S. Havulinna, et
al. “Taxonomic signatures of cause-specific mortality risk in human gut
microbiome.” In: *Nature Communications* 12 (2021), p. 2671. DOI:
10.1038/s41467-021-22962-y.

\[96\] V. K. Sarhadi, L. Lahti, I. Scheinin, et al. “Targeted
Resequencing of 9p in Acute Lymphoblastic Leukemia Yields Concordant
Results with Array CGH and Reveals Novel Genomic Alterations.” In:
*Genomics* 102.3 (sept.. 2013), pp. 182-188. DOI:
10.1016/j.ygeno.2013.01.001.

\[97\] V. Sarhadi, L. Lahti, F. Saberi, et al. “Gut Microbiota and Host
Gene Mutations in Colorectal Cancer Patients and Controls of Iranian and
Finnish Origin.” In: *Anticancer Research* 40.3 (mars. 2020), pp.
1325-1334. DOI: 10.21873/anticanres.14074.

\[98\] V. Sarhadi, L. Lahti, I. Scheinin, et al. “Copy Number
Alterations and Neoplasia Specific Mutations in MELK, PDCD1LG2, TLN1,
and PAX5 at 9p in Different Neoplasias.” In: *Genes, Chromosomes and
Cancer* 53.7 (juil.. 2014), pp. 579-588. DOI: 10.1002/gcc.22168.

\[99\] V. Sarhadi, B. Mathew, A. K. A, et al. “Gut microbiota of
patients with different subtypes of gastric cancer and gastrointestinal
stromal tumors.” In: *Gut Pathogens* 13 (11 2021), pp. 1-9. DOI:
10.1186/s13099-021-00403-x.

\[100\] S. Shetty, F. Hugenholtz, L. Lahti, et al. “Intestinal
microbiome landscaping: Insight in community assemblage and implications
for microbial modulation strategies.” In: *FEMS Microbiology Reviews* 41
(2 mars. 2017), pp. 182-199. DOI: 10.1093/femsre/fuw045.

\[101\] S. Shetty and L. Lahti. “Microbiome data science.” In: *Journal
of Biosciences* 44 (oct.. 2019). Preprint openly available at:
<https://openresearchlabs.github.io/publications/papers/2019-Shetty-MDS.pdf>,
p. 115. DOI: 10.1007/s12038-019-9930-2.

\[102\] J. Sinkkonen, S. Kaski, J. N. ä, et al. *Associative Clustering
(AC): Technical Details*. Tech. rep. A84. Publications in Computer and
Information Science. Espoo, Finland: Helsinki University of Technology,
avr.. 2005.

\[103\] J. Sinkkonen, J. Nikkilä, L. Lahti, et al. “Associative
Clustering.” In: *Proceedings of the ECML’04, 15th European Conference
on Machine Learning*. Ed. by Boulicaut, Esposito, Giannotti and
Pedreschi. Berlin: Springer, 2004, pp. 396-406. &lt;URL:
<http://www.cis.hut.fi/projects/mi/abstracts/ecml04.html>&gt;.

\[104\] J. Sinkkonen, J. Nikkilä, L. Lahti, et al. *Associative
Clustering by Maximizing a Bayes Factor*. Tech. rep. A68. Espoo,
Finland: Helsinki University of Technology and Laboratory of Computer
and Information Science, juin. 2003.

\[105\] M. Stolaki, M. Minekus, K. Venema, et al. “Microbial communities
in a dynamic it in vitro model for the human ileum resemble the human
ileal microbiota.” In: *FEMS Microbiology Ecology* (août. 2019), p.
fiz096. DOI: 10.1093/femsec/fiz096.

\[106\] J. Su, C. Ekman, N. Oskolkov, et al. “A novel atlas of gene
expression in human skeletal muscle reveals molecular changes associated
with aging.” In: *Skeletal Muscle* 5.35 (oct.. 2015). DOI:
10.1186/s13395-015-0059-1.

\[107\] M. Tolonen, M. J. Hill, A. Z. Ijaz, et al. “Data Visualization
in Enlightenment Literature and Culture.” In: *Data Visualization in
Enlightenment Literature and Culture*. *Data Visualization in
Enlightenment Literature and Culture*. Ed. by I. Baird. Palgrave
Macmillan, Cham., 2021. Chap. Examining the Early Modern Canon: The
English Short Title Catalogue and Large-Scale Patterns of Cultural
Production. DOI: 10.1007/978-3-030-54913-8\_3.

\[108\] M. Tolonen, N. IlomÃ¤ki, H. Roivainen, et al. “Printing in a
Periphery: a Quantitative Study of Finnish Knowledge Production,
1640-1828.” In: *Digital Humanities 2016: Conference Abstracts*. Krakow,
Poland: Jagiellonian University & Pedagogical University, KrakÃƒÂ³w,
juil.. 2016, pp. 383-385. &lt;URL:
<http://dh2016.adho.org/abstracts/9>&gt;.

\[109\] M. Tolonen and L. Lahti. “Aatehistoria ja digitaalisten
aineistojen mahdollisuudet.” Finnish. In: *Ennen & Nyt 2* 2 (août.
2015). &lt;URL:
<http://www.ennenjanyt.net/2015/08/aatehistoria-ja-digitaalisten-aineistojen-mahdollisuudet>&gt;.

\[110\] M. Tolonen and L. Lahti. “Digitaaliset ihmistieteet ja
historiantutkimus.” Finnish. In: *Menneisyyden rakentajat* (2018). Ed.
by M. Hannikainen, M. Danielsbacka and T. Tepora. (in Finnish). In:
Hannikainen, MO and Danielsbacka, M and Tepora, T (eds.). Menneisyyden
rakentajat. Gaudeamus, Helsinki, 2018. &lt;URL:
<https://www.gaudeamus.fi/menneisyyden-rakentajat/>&gt;.

\[111\] M. Tolonen, L. Lahti, H. Roivainen, et al. “A Quantitative
Approach to Book-Printing in Sweden and Finland, 1640â€“1828.” In:
*Historical Methods: A Journal of Quantitative and Interdisciplinary
History* 52 (1 2019), pp. 57-78. DOI: 10.1080/01615440.2018.1526657.

\[112\] M. Tolonen, J. Marjanen, H. Roivainen, et al. “Scaling up
bibliographic data science.” In: *Proceedings of the Digital Humanities
in the Nordics (DHN2019)*. Ed. by C. Navarretta, M. Agirrezabal and B.
Maegaard. Copenhagen, mars. 2019.

\[113\] U. of Turku. “Turun yliopiston avoimen tutkimuksen politiikka /
Open Research Policy.” In: *unknown* (mai. 2018). (in Finnish). &lt;URL:
<https://www.utu.fi/sites/default/files/public%3A//media/file/ty-avoimen-tutkimuksen-politiikka-fi.pdf>&gt;.

\[114\] V. Vaara, A. Ijaz, I. Tiihonen, et al. “The Emerging Paradigm of
Bibliographic Data Science.” In: *Proceedings of the Digital Humanities
(DH2019)*. In press.. juil.. 2019.

\[115\] F. C. Vaura, V. V. Salomaa, I. M. Kantola, et al. “Unsupervised
hierarchical clustering identifies a metabolically challenged subgroup
of hypertensive individuals.” In: *Journal of Clinical Hypertension*
22.9 (aoÃ»t. 2020), pp. 1546-1553. DOI: 10.1111/jch.13984.

\[116\] O. Youssef, L. Lahti, A. Kokkola, et al. “Stool Microbiota
Composition Differs in Patients with Stomach, Colon, and Rectal
Neoplasms.” In: *Digestive Diseases and Sciences* 63.11 (juil.. 2018),
pp. 2950-2958. DOI: 10.1007/s10620-018-5190-5.

\[117\] Q. Zhu, S. Huang, A. Gonzalez, et al. “OGUs enable effective,
phylogeny-aware analysis of even shallow metagenome community
structures.” In: *bioRxiv* (2021). DOI: 10.1101/2021.04.04.438427.
eprint:
<https://www.biorxiv.org/content/early/2021/04/06/2021.04.04.438427.full.pdf>.
&lt;URL:
<https://www.biorxiv.org/content/early/2021/04/06/2021.04.04.438427>&gt;.
