---
description: Code resources
keywords: code
title: Code
---


# Open research data and software

[![Research software impact](http://depsy.org/api/person/333684/badge.svg)](http://depsy.org/person/333684). Mature open source projects are listed below. 


### Statistical ecology and machine learning

**[Microbiome bioinformatics software and tutorials](http://microbiome.github.io)**

**[earlywarnings](http://www.early-warning-signals.org/)** Methods for identification of critical transitions between ecosystem states from time series data (CRAN). Co-developer. [CRAN](http://cran.r-project.org/web/packages/earlywarnings/index.html)/[Github](https://github.com/earlywarningtoolbox/earlywarnings-R/tree/master/earlywarnings). _WICI Data Challenge 2013 runner-up / Waterloo Institute for Complex Systems and Innovation._

**[DMT](http://dmt.r-forge.r-project.org)** Dependency Modeling Toolkit. Probabilistic tools for dependency analysis between multiple data sources (R/CRAN). Probabilistic PCA, factor analysis, CCA, regularized variants, dependency-based dimensionality reduction etc. _ICML/MLOSS workshop, Israel 2010_.



### Bioinformatics

**[netresponse](http://bioconductor.org/packages/release/bioc/html/netresponse.html)** Modeling context-specific activation patterns in genome-wide interaction networks (R/Matlab). Originally applied to study transcriptional responses in genome-scale interaction networks across organism-wide collections of gene expression data. [doi:10.18129/B9.BIOC.NETRESPONSE](https://doi.org/10.18129/B9.BIOC.NETRESPONSE)

**[RPA](http://bioconductor.org/packages/release/bioc/html/RPA.html)** (R/Bioconductor). Scalable probabilistic method for preprocessing short oligo microarray data. [_NAR_ 2013](http://nar.oxfordjournals.org/content/early/2013/04/05/nar.gkt229.abstract). [doi:10.18129/B9.BIOC.RPA](https://doi.org/10.18129/B9.BIOC.RPA)

**[intcomp](http://intcomp.r-forge.r-project.org)** Benchmarking for integrative cancer gene discovery algorithms. [_Briefings in Bioinformatics_ 2012](http://bib.oxfordjournals.org/content/early/2012/03/21/bib.bbs005.abstract)

**[pint](https://github.com/antagomir/pint)** Probabilistic data integration for DNA/RNA data in functional genomics (R/Bioc). [_MLSP_ 2009](http://arxiv.org/abs/1101.5919).


### Digital Humanities and Computational Social Science

**[rOpenGov](https://ropengov.org)** R package ecosystem for open government data analytics (R/GitHub). introduced in NIPS Machine Learning Open Source Software workshop 2013.

**[COMHIS](http://helsinki.fi/computational-history)** Helsinki Computational History Group data analytics infrastructure.


## Data

[Full agreement texts for academic publisher agreements](http://finelib.fi/negotiations/negotiations/). The agreements were released by the FinELib consortium of research libraries following our FOI request in April 2018.

[Scientific journal subscription costs in Finland 2010-2017; MoE/ATT](http://ropengov.github.io/r/2018/12/05/FOI/). The data set was released by Finnish Ministry of Education (Open Science and Research Initiative) following my Freedom of Information Request ([summary](http://ropengov.github.io/r/2016/06/10/FOI/)). Finland became the first country to systematically collect and release this information.

[Human Gut Microbiota Atlas (Data Dryad)](http://doi.org/10.5061/dryad.pk75d); Genus level profiling of the human gut microbiota for 1006 western adults; [Lahti et al. _Nature Communications_ 5:4344, 2014](http://www.nature.com/ncomms/2014/140708/ncomms5344/full/ncomms5344.html)

[Gut microbiota profiling in African-American diet swap experiment](http://dx.doi.org/10.5061/dryad.1mn1n); Data Dryad; [Oâ€™Keefe et al. _Nature Communications_ 6:6342, 2015](http://www.nature.com/ncomms/2015/150428/ncomms7342/full/ncomms7342.html)

[Probiotics intervention data with high-throughput profiling of the gut microbiome and serum lipidome](https://github.com/microbiome/microbiome/blob/master/data/peerj32.rda) (RData format); from Associations between the human intestinal microbiota, Lactobacillus rhamnosus GG and serum lipids indicated by integrated analysis of high-throughput profiling data. [Lahti et al. PeerJ 1:e32, 2013](http://dx.doi.org/10.7717/peerj.32) 



