<h1 id="references">References</h1>

<p>Shetty, S., &amp; Lahti, L. (2019). Microbiome data science. <i>Journal of Biosciences</i>, <i>44</i>, 115. https://doi.org/10.1007/s12038-019-9930-2</p>

<p>Marjanen, J., Vaara, V., Kanner, A., Roivainen, H., Mäkelä, E., Lahti, L., &amp; Tolonen, M. (2019). A National Public Sphere? Analyzing the Language, Location, and Form of Newspapers in Finland, 1771–1917. <i>Journal of European Periodical Studies</i>, <i>4</i>(1). https://doi.org/10.21825/jeps.v4i1.10483</p>

<p>Ijaz, A., Tolonen, M., &amp; Lahti, L. (2019). Analytical determination of editions from bibliographic metadata. In <i>Proceedings of the Research Data in the Humanities Conference</i>. Oulu.</p>

<p>Lahti, L., Vaara, V., Marjanen, J., &amp; Tolonen, M. (2019). Best practices in bibliographic data science. In <i>Proceedings of the Research Data in the Humanities Conference</i>. Oulu.</p>

<p>Gut microbiota composition is associated with temperament traits in infants. (2019). <i>Brain Behavior and Immunity</i>. Bioscience. https://doi.org/10.1016/j.bbi.2019.05.035</p>

<p>K, K. T., F, F. J., M, M. R., L, L. L., &amp; Molinari, G. (2019). Microbiome yarns: The Global Phenotype-Genotype Survey. Episode III: importance of microbiota diversification for microbiome function and biome health. <i>Microbial Biotechnology</i>. Misc. https://doi.org/10.1111/1751-7915.13406</p>

<p>Stolaki, M., Minekus, M., Venema, K., Lahti, L., Smid, E. J., Kleerebezem, M., &amp; Zoetendal, E. G. (2019). Microbial Communities in a Dynamic In Vitro Model for the Human Ileum Resemble the Human Ileal Microbiota. <i>FEMS Microbiology Ecology</i>, (8). https://doi.org/10.1093/femsec/fiz096</p>

<p>Tolonen, M., Ilomäki, N., Roivainen, H., &amp; Lahti, L. (2016). Printing in a Periphery: a Quantitative Study of Finnish Knowledge Production, 1640-1828. In <i>Digital Humanities 2016: Conference Abstracts</i> (pp. 383–385). Krakow, Poland: Jagiellonian University &amp; Pedagogical University, Kraków. Retrieved from http://dh2016.adho.org/abstracts/9</p>

<p>Vaara, V., Ijaz, A., Tiihonen, I., Hengchen, S., Kanner, A., Säily, T., &amp; Lahti, L. (2019). The Emerging Paradigm of Bibliographic Data Science. In <i>Proceedings of the Digital Humanities (DH2019)</i>.</p>

<p>Ijaz, A., Roivainen, H., &amp; Lahti, L. (2019). Analytical Edition Detection In Bibliographic Metadata. In <i>Proceedings of the Digital Humanities (DH2019)</i>.</p>

<p>Hill, M. J., Vaara, V., Säily, T., Lahti, L., &amp; Tolonen, M. (2019). Reconstructing intellectual networks: from the ESTC’s bibliographic metadata to historical material. In C. Navarretta, M. Agirrezabal, &amp; B. Maegaard (Eds.), <i>Proceedings of the Digital Humanities in the Nordics (DHN2019)</i>. Copenhagen. Retrieved from https://cst.dk/DHN2019Pro/papers/DHN2019_hill.pdf</p>

<p>Mäkela, E., Tolonen, M., Marjanen, J., Kanner, A., Vaara, V., &amp; Lahti, L. (2019). Interdisciplinary collaboration in studying newspaper materiality. In C. Navarretta, M. Agirrezabal, &amp; B. Maegaard (Eds.), <i>Proceedings of the Twin Talks workshop. Digital Humanities in the Nordics (DHN2019)</i>. Copenhagen. Retrieved from http://ceur-ws.org/Vol-2365/07-TwinTalks-DHN2019_paper_7.pdf</p>

<p>Tolonen, M., Marjanen, J., Roivainen, H., &amp; Lahti, L. (2019). Scaling up bibliographic data science. In C. Navarretta, M. Agirrezabal, &amp; B. Maegaard (Eds.), <i>Proceedings of the Digital Humanities in the Nordics (DHN2019)</i>. Copenhagen.</p>

<p>Lahti, L., Marjanen, J., Roivainen, H., &amp; Tolonen, M. (2019). Bibliographic Data Science and the History of the Book (c. 1500–1800). <i>Cataloging &amp; Classification Quarterly</i>, 1–19. https://doi.org/10.1080/01639374.2018.1543747</p>

<p>Shetty, S., &amp; Lahti, L. (2018). Microbiome Data Science. Bioscience.</p>

<p>Tolonen, M., Lahti, L., Roivainen, H., &amp; Marjanen, J. (2019). A Quantitative Approach to Book-Printing in Sweden and Finland, 1640–1828. <i>Historical Methods: A Journal of Quantitative and Interdisciplinary History</i>, <i>52</i>(1), 57–78. https://doi.org/10.1080/01615440.2018.1526657</p>

<p>Lahti, L. (2018). Open Data Science. In <i>Advances in Intelligent Data Analysis XVII. Lecture Notes in Computer Science 11191.</i> (Vol. 11191). India: Springer Nature.</p>

<p>Laitinen, V., &amp; Lahti, L. (2018). A hierarchical Ornstein-Uhlenbeck model for stochastic time series analysis. In <i>Advances in Intelligent Data Analysis XVII. Lecture Notes in Computer Science 11191.</i> India: Springer. Retrieved from https://github.com/velait/OUP</p>

<p>Gonze, D., Coyte, K. Z., Lahti, L., &amp; Faust, K. (2018). Microbial communities as dynamical systems. <i>Current Opinion in Microbiology</i>, <i>44</i>, 41–49. https://doi.org/10.1016/j.mib.2018.07.004</p>

<p>Youssef, O., Lahti, L., Kokkola, A., Karla, T., Tikkanen, M., Ehsan, H., … Sarhadi, V. (2018). Stool Microbiota Composition Differs in Patients with Stomach, Colon, and Rectal Neoplasms. <i>Digestive Diseases and Sciences</i>, <i>63</i>(11), 2950–2958. https://doi.org/10.1007/s10620-018-5190-5</p>

<p>of Turku, U. (2018). Turun yliopiston avoimen tutkimuksen politiikka / Open Research Policy. Openscience. Retrieved from https://www.utu.fi/sites/default/files/public%3A//media/file/ty-avoimen-tutkimuksen-politiikka-fi.pdf</p>

<p>Faust, K., Bauchinger, F., Laroche, B., de Buyl, S., Lahti, L., Washburne, A. D., … Widder, S. (2018). Signatures of ecological processes in microbial community time series. <i>Microbiome</i>, <i>6</i>(120). https://doi.org/10.1186/s40168-018-0496-2</p>

<p>Aatsinki, A.-K., Uusitupa, H.-M., Munukka, E., Pesonen, H., Rintala, A., Pietilä, S., … Karlsson, H. (2018). Gut Microbiota Composition in Mid-Pregnancy Is Associated with Gestational Weight Gain but Not Prepregnancy Body Mass Index. <i>Journal of Womens Health</i>. Bioscience. https://doi.org/10.1089/jwh.2017.6488</p>

<p>Arani, B. M. S., Mahmoudi, M., Lahti, L., González, J., &amp; Wit, E. C. (2018). Stability estimation of autoregulated genes under Michaelis-Menten type kinetics. <i>Physical Review E</i>, <i>97</i>(6), 062407. https://doi.org/10.1103/PhysRevE.97.062407</p>

<p>Tolonen, M., &amp; Lahti, L. (2018). Digitaaliset ihmistieteet ja historiantutkimus. Dh. Retrieved from https://www.gaudeamus.fi/menneisyyden-rakentajat/</p>

<p>Lahti, L. (2007). A brief overview on the BioPAX and SBML standards. arXiv. Retrieved from https://arxiv.org/abs/1109.4919</p>

<p>Lahti, L. (2001). Julian joukko iteraatioden valuma-altaan reunana. Retrieved from https://github.com/antagomir/thesis/tree/master/julia01</p>

<p>Lahti, L. (2002). Itsesimilaaristen joukkojen Hausdorffin dimension määrittäminen. Retrieved from https://github.com/antagomir/thesis/tree/master/hausdorff02</p>

<p>Hill, M. J., Kanner, A., Marjanen, J., Vaara, V., Mäkelä, E., Lahti, L., &amp; Tolonen, M. (2018). Spheres of “public” in eighteenth-century Britain. In <i>Proceedings of the Digital Humanities in the Nordics Conference</i>. Helsinki, Finland.</p>

<p>Björk, A., Paavolainen, J.-M., Ropponen, T., Laakso, M., &amp; Lahti, L. (2018). <i>Opening academic publishing: development and application of systematic evaluation criteria</i>. Open Science and Research Initiative (ATT). Retrieved from http://urn.fi/URN:NBN:fi-fe201802123334</p>

<p>Buelow, E., González, T. d.J. B., Fuentes, S., de Steenhuijsen Piters, W. A. A., Lahti, L., Bayjanov, J. R., … van Schaik, W. (2017). Comparative gut microbiota and resistome profiling of intensive care patients receiving selective digestive tract decontamination and healthy subjects. <i>Microbiome</i>, <i>5</i>(1), 88. https://doi.org/10.1186/s40168-017-0309-z</p>

<p>Lahti, L., da Silva, F., Laine, M. P., Lähteenoja, V., &amp; Tolonen, M. (2017). Alchemy &amp; algorithms: perspectives on the philosophy and history of open science. <i>RIO Journal</i>, <i>3</i>, e13593. https://doi.org/10.3897/rio.3.e13593</p>

<p>Faust, K., Gonze, D., Lahti, L., &amp; Raes, J. (2017). Multi-stability and the origin of microbial community types. <i>ISME Journal</i>, <i>11</i>, 2159–2166. https://doi.org/10.1038/ismej.2017.60</p>

<p>Tolonen, M., &amp; Lahti, L. (2015). Aatehistoria ja digitaalisten aineistojen mahdollisuudet. <i>Ennen &amp; Nyt 2</i>, <i>2</i>. Dh. Retrieved from http://www.ennenjanyt.net/2015/08/aatehistoria-ja-digitaalisten-aineistojen-mahdollisuudet</p>

<p>Lahti, L., Ilomäki, N., &amp; Tolonen, M. (2015). A quantitative study of history in the English short-title catalogue (ESTC) 1470-1800. <i>LIBER Quarterly</i>, <i>25</i>(2), 87–116. https://doi.org/10.18352/lq.10112</p>

<p>Ritari, J., Salojärvi, J., Lahti, L., &amp; de Vos, W. M. (2015). Improved taxonomic assignment of human intestinal 16S rRNA sequences by a dedicated reference database. <i>BMC Genomics</i>, <i>16</i>, 1056. https://doi.org/10.1186/s12864-015-2265-y</p>

<p>Laine, H., Lahti, L., Lehto, A., Ollila, S., &amp; Miettinen, M. (2015). Beyond Open Access - The Changing Culture of Producing and Disseminating Scientific Knowledge. In <i>Proceedings of the 19th International Academic Mindtrek Conference in Tampere, Finland, September 22-24</i>. New York, NY, USA: ACM. Retrieved from http://dl.acm.org/citation.cfm?id=2818187</p>

<p>Su, J., Ekman, C., Oskolkov, N., Lahti, L., Ström, K., Brazma, A., … and Ola Hansson. (2015). A novel atlas of gene expression in human skeletal muscle reveals molecular changes associated with aging. <i>Skeletal Muscle</i>, <i>5</i>(35). https://doi.org/10.1186/s13395-015-0059-1</p>

<p>Leo Lahti, J. L., Juuso Parkkinen, &amp; Kainu, M. (2013). rOpenGov: open source ecosystem for computational social sciences and digital humanities. software. Retrieved from http://ropengov.github.io</p>

<p>Atashgahi, S., Aydin, R., Dimitrov, M. R., Sipkema, D., Hamonts, K., Lahti, L., … Smidt, H. (2015). Impact of a wastewater treatment plant on microbial community composition and function in a hyporheic zone of a eutrophic river. <i>Scientific Reports</i>, <i>5</i>(17284). https://doi.org/10.1038/srep17284</p>

<p>Faust, K., Lahti, L., Gonze, D., de Vos, W. M., &amp; Raes, J. (2015). Metagenomics meets time series analysis: unraveling microbial community dynamics. <i>Current Opinion in Microbiology</i>, <i>25</i>, 56–66. https://doi.org/10.1016/j.mib.2015.04.004</p>

<p>O’Keefe, S. J. D., Li, J. V., Lahti, L., Ou, J., Carbonero, F., Mohammed, K., … Zoetendal, E. G. (2015). Fat, Fiber and Cancer Risk in African, Americans and Rural Africans. <i>Nature Communications</i>, <i>6</i>, 6342. https://doi.org/10.1038/ncomms7342</p>

<p>Aleksic, J., Alexa, A., Attwood, T. K., Hong, N. C., Dahlö, M., Davey, R., … Vieira, B. M. (2015). The Open Science Peer Review Oath. <i>F1000Research</i>, <i>3</i>, 271. https://doi.org/10.12688/f1000research.5686.2</p>

<p>Niini, T., Scheinin, I., Lahti, L., Savola, S., Mertens, F., Hollmén, J., … Knuutila, S. (2012). Homozygous Deletions of Cadherin Genes in Chondrosarcoma – an Array CGH Study. <i>Cancer Genetics</i>, <i>205</i>(11), 588–593. https://doi.org/10.1016/j.cancergen.2012.09.007</p>

<p>Salonen, A., Salojärvi, J., Lahti, L., &amp; and Willem M. de Vos. (2012). The adult intestinal core microbiota is determined by analysis depth and health status. <i>Clinical Microbiology and Infection</i>, <i>18</i>(Suppl. 4), 16–20. https://doi.org/10.1111/j.1469-0691.2012.03855.x</p>

<p>Mosakhani, N., Lahti, L., Borze, I., Karjalainen-Lindsberg, M.-L., Sundstrom, J., Ristamaki, R., … and Virinder Kaur Sarhadi. (2012). MicroRNA profiling predicts survival in anti-EGFR treated chemorefractory metastatic colorectal cancer patients with wild-type KRAS and BRAF. <i>Cancer Genetics</i>, <i>205</i>(11), 545–551. https://doi.org/10.1016/j.cancergen.2012.08.003</p>

<p>Lahti, L., Schäfer, M., Klein, H.-U., Bicciato, S., &amp; Dugas, M. (2013). Cancer gene prioritization by integrative analysis of mRNA expression and DNA copy number data: a comparative review. <i>Briefings in Bioinformatics</i>, <i>14</i>(1), 27–35. https://doi.org/10.1093/bib/bbs005</p>

<p>Lahti, L., Salonen, A., Kekkonen, R. A., Salojärvi, J., Jalanka-Tuovinen, J., Palva, A., … de Vos, W. M. (2013). Associations between the human intestinal microbiota, Lactobacillus rhamnosus GG and serum lipids indicated by integrated analysis of high-throughput profiling data. <i>PeerJ</i>, <i>1</i>, e32. https://doi.org/10.7717/peerj.32</p>

<p>Sarhadi, V. K., Lahti, L., Scheinin, I., Tyybäkinoja, A., Savola, S., Usvasalo, A., … Knuutila, S. (2013). Targeted Resequencing of 9p in Acute Lymphoblastic Leukemia Yields Concordant Results with Array CGH and Reveals Novel Genomic Alterations. <i>Genomics</i>, <i>102</i>(3), 182–188. https://doi.org/10.1016/j.ygeno.2013.01.001</p>

<p>Mosakhani, N., Sarhadi, V. K., Usvasalo, A., Karjalainen-Lindsberg, M. L., Lahti, L., Tuononen, K., … Knuutila, S. (2012). MicroRNA profiling in pediatric acute lymphoblastic leukemia: novel prognostic tools. <i>Leukemia &amp; Lymphoma</i>, <i>53</i>(12), 2517–2520. https://doi.org/10.3109/10428194.2012.685731</p>

<p>Harris, K., Parsons, T. L., Ijaz, U. Z., Lahti, L., Holmes, I., &amp; Quince, C. (2017). Linking statistical and ecological theory: Hubbell’s unified neutral theory of biodiversity as a hierarchical Dirichlet process. <i>Proceedings of the IEEE</i>, <i>105</i>(3), 516–529. https://doi.org/10.1109/JPROC.2015.2428213</p>

<p>Alneberg, J., Bjarnason, B. S., de Bruijn, I., Schirmer, M., Quick, J., Ijaz, U. Z., … Quince, C. (2014). Binning metagenomic contigs by coverage and composition. <i>Nature Methods</i>, <i>11</i>(1144–1146). https://doi.org/10.1038/nmeth.3103</p>

<p>Lahti, L., Salojärvi, J., Salonen, A., Scheffer, M., &amp; de Vos, W. M. (2014). Tipping elements in the human intestinal ecosystem. <i>Nature Communications</i>, <i>5</i>, 4344. https://doi.org/10.1038/ncomms5344</p>

<p>Salonen, A., Lahti, L., Salojärvi, J., Holtrop, G., Korpela, K., Duncan, S., … de Vos, W. M. (2014). Impact of diet and individual variation on intestinal microbiota composition and fermentation products in obese men. <i>ISME Journal</i>, <i>8</i>, 2218–2230. https://doi.org/10.1038/ismej.2014.63</p>

<p>Lahti, L. (2013). Intestinal microbiota, individuality and health. In: Computational Methods Aiding Early-Stage Drug Design (Dagstuhl Seminar 13212). <i>Dagstuhl Reports</i>, <i>3</i>(5), 78–94. https://doi.org/10.4230/DagRep.3.5.78</p>

<p>Lahti, L., Elo, L. L., Aittokallio, T., &amp; Kaski, S. (2011). Probabilistic Analysis of Probe Reliability in Differential Gene Expression Studies with Short Oligonucleotide Arrays. <i>IEEE/ACM Transactions on Computational Biology and Bioinformatics</i>, <i>8</i>(1), 217–225. https://doi.org/10.1109/TCBB.2009.38</p>

<p>Lahti, L., Lehtomäki, J., Parkkinen, J., &amp; others. (blog). Louhos blog. Retrieved from http://louhos.github.io/</p>

<p>Lahti, L., Lehtomäki, J., Kainu, M., Parkkinen, J., &amp; others. (blog). rOpenGov blog. Retrieved from http://ropengov.github.io/</p>

<p>Lahti, L., Schäfer, M., Klein, H.-U., Bicciato, S., &amp; Dugas, M. (2011). intcomp: a benchmarking pipeline for integrative cancer gene detection algorithms. software. Retrieved from http://intcomp.r-forge.r-project.org/</p>

<p>Jalanka-Tuovinen, J., Salonen, A., Nikkilä, J., Immonen, O., Kekkonen, R., Lahti, L., … de Vos, W. M. (2011). Intestinal microbiota in healthy adults: Temporal analysis reveals individual and common core and relation to intestinal symptoms. <i>PLoS One</i>, <i>6</i>(7), e23035. Retrieved from 10.1371/journal.pone.0023035</p>

<p>Louhimo, R., Aittomäki, V., Faisal, A., Laakso, M., Chen, P., Ovaska, K., … Hautaniemi., S. (2014). Systematic Use of Computational Methods Allows Stratifying Treatment Responders in Glioblastoma Multiforme. <i>Systems Biomedicine (Special Issue)</i>, <i>1</i>(2), 130–136. https://doi.org/10.4161/sysb.28904</p>

<p>Lahti, L., &amp; Kaski, S. (2011). Probabilistic dependency models for data integration in functional genomics.</p>

<p>Nymark, P., Guled, M., Borze, I., Faisal, A., Lahti, L., Salmenkivi, K., … Knuutila, S. (2011). Integrative Analysis of microRNA, mRNA and aCGH Data Reveals Asbestos- and Histology-Related Changes in Lung Cancer. <i>Genes, Chromosomes and Cancer</i>, <i>50</i>(8), 585–597. https://doi.org/10.1002/gcc.20880</p>

<p>Niini, T., Lahti, L., Michelacci, F., Ninomiya, S., Hattinger, C. M., Guled, M., … Knuutila, S. (2011). Array Comparative Genomic Hybridization Reveals Frequent Alterations of G1/S Checkpoint Genes in Undifferentiated Pleomorphic Sarcoma of Bone. <i>Genes, Chromosomes and Cancer</i>, <i>50</i>(5), 291–306. https://doi.org/10.1002/gcc.20851</p>

<p>Elo, L. L., Lahti, L., Skottman, H., Kyläniemi, M., Lahesmaa, R., &amp; Aittokallio, T. (2005). Integrating probe-level expression changes across generations of Affymetrix arrays. <i>Nucleic Acids Research</i>, <i>33</i>(22), e193. https://doi.org/10.1093/nar/gni193</p>

<p>Kaski, S., Nikkilä, J., Sinkkonen, J., Lahti, L., Knuuttila, J., &amp; Roos, C. (2005). Associative clustering for exploring dependencies between functional genomics data sets. <i>IEEE/ACM Transactions on Computational Biology and Bioinformatics</i>, <i>2</i>(3), 203–216. https://doi.org/10.1109/TCBB.2005.32</p>

<p>Lahti, L. (2003). <i>Comparative functional genome analysis using associative clustering</i> (Master’s thesis). Helsinki University of Technology. Retrieved from https://github.com/antagomir/thesis/tree/master/msc03</p>

<p>Lahti, L. (2009). Concerning Blackburn’s quasi-realism in securing moral discussion. B.Sc. thesis. University of Helsinki, Faculty of Political Sciences.</p>

<p>Nymark, P., Lindholm, P. M., Korpela, M. V., Lahti, L., Ruosaari, S., Kaski, S., … Knuutila, S. (2007). Gene Expression Profiles in Asbestos-exposed Epithelial and Mesothelial Lung Cell Lines. <i>BMC Genomics</i>, <i>8</i>(62). https://doi.org/10.1186/1471-2164-8-62</p>

<p>Sinkkonen, J., Kaski, S., Nikkilä, J., &amp; Lahti, L. (2005). <i>Associative Clustering (AC): Technical Details</i> (No. A84). Espoo, Finland: Helsinki University of Technology.</p>

<p>Sinkkonen, J., Nikkilä, J., Lahti, L., &amp; Kaski, S. (2004). Associative Clustering. In Boulicaut, Esposito, Giannotti, &amp; Pedreschi (Eds.), <i>Proceedings of the ECML’04, 15th European Conference on Machine Learning</i> (pp. 396–406). Berlin: Springer. Retrieved from http://www.cis.hut.fi/projects/mi/abstracts/ecml04.html</p>

<p>Sinkkonen, J., Nikkilä, J., Lahti, L., &amp; Kaski, S. (2003). <i>Associative Clustering by Maximizing a Bayes Factor</i> (No. A68). Espoo, Finland: Helsinki University of Technology, Laboratory of Computer and Information Science.</p>

<p>Lahti, L., Huovari, J., Kainu, M., &amp; Biecek, P. (2017). Retrieval and Analysis of Eurostat Open Data with the eurostat Package. <i>The R Journal</i>, <i>9</i>(1), 385–392. Retrieved from https://journal.r-project.org/archive/2017/RJ-2017-019/index.html</p>

<p>Shetty, S. A., Hugenholtz, F., Lahti, L., Smidt, H., de Vos, W. M., &amp; Danchin, A. (2017). Intestinal microbiome landscaping: Insight in community assemblage and implications for microbial modulation strategies. <i>FEMS Microbiology Reviews</i>, <i>41</i>(2), fuw045. https://doi.org/10.1093/femsre/fuw045</p>

<p>Sarhadi, V., Lahti, L., Scheinin, I., Ellonen, P., Kettunen, E., Serra, M., … Knuutila, S. (2014). Copy Number Alterations and Neoplasia Specific Mutations in MELK, PDCD1LG2, TLN1, and PAX5 at 9p in Different Neoplasias. <i>Genes, Chromosomes and Cancer</i>, <i>53</i>(7), 579–588. https://doi.org/10.1002/gcc.22168</p>

<p>Lahti, L., Torrente, A., Elo, L. L., Brazma, A., &amp; Rung, J. (2013). A fully scalable online pre-processing algorithm for short oligonucleotide microarray atlases. <i>Nucleic Acids Research</i>, <i>41</i>(10), e110. https://doi.org/10.1093/nar/gkt229</p>

<p>Faisal, A., Louhimo, R., Lahti, L., Hautaniemi, S., &amp; Kaski, S. (2011). Biomarker discovery via dependency analysis of multi-view functional genomics data. In <i>NIPS 2011 workshop From Statistical Genetics to Predictive Models in Personalized Medicine</i>.</p>

<p>Lahti, L. (2010). <i>Probabilistic analysis of the human transcriptome with side information</i> (PhD thesis). Aalto University School of Science and Technology, Faculty of Information and Natural Sciences, Department of Information and Computer Science, Espoo. Retrieved from http://lib.tkk.fi/Diss/2010/isbn9789526033686/</p>

<p>Lahti, L., Knuuttila, J. E. A., &amp; Kaski, S. (2010). Global modeling of transcriptional responses in interaction networks. <i>Bioinformatics</i>, <i>26</i>(21), 2713–2720. https://doi.org/10.1093/bioinformatics/btq500</p>

<p>Mosakhani, N., Guled, M., Lahti, L., Borze, I., Forsman, M., Ryhänen, J., &amp; Knuutila, S. (2010). Unique microRNA profile in Dupuytren’s contracture supports deregulation of β-catenin pathway. <i>Modern Pathology</i>, <i>23</i>, 1544–1522. https://doi.org/10.1038/modpathol.2010.146</p>

<p>Borze, I., Guled, M., Musse, S., Raunio, A., Elonen, E., Saarinen-Pihkala, U., … Knuutila., S. (2011). MicroRNA microarrays on archive bone marrow core biopsies of leukemias - method validation. <i>Leukemia Research</i>, <i>35</i>(2), 188–195. https://doi.org/10.1016/j.leukres.2010.08.005</p>

<p>Lahti, L. opencomp blog. Retrieved from http://antagomir.wordpress.com/</p>

<p>Huovilainen, O.-P., &amp; Lahti, L. (2010). Pairwise integration of functional genomics data. software. Retrieved from http://bioconductor.org/packages/devel/bioc/html/pint.html</p>

<p>Lahti, L., Tripathi, A., &amp; Huovilainen, O.-P. (2010). Dependency modeling toolkit. ICML workshop. Retrieved from http://dmt.r-forge.r-project.org/</p>

<p>Lahti, L., Gusmao, A., &amp; Huovilainen, O.-P. (2010). NetResponse (functional network analysis). software. https://doi.org/10.18129/B9.BIOC.NETRESPONSE</p>

<p>Lahti, L. (2009). RPA: probe reliability and differential gene expression analysis. software. https://doi.org/10.18129/B9.BIOC.RPA</p>

<p>Guled, M., Lahti, L., Lindholm, P. M., Salmenkivi, K., Bagwan, I., Nicholson, A. G., &amp; Knuutila, S. (2009). CDKN2A, NF2 and JUN Are Dysregulated Among Other Genes by miRNAs in Malignant Mesothelioma - a miRNA Microarray Analysis. <i>Genes, Chromosomes and Cancer</i>, <i>48</i>(7), 615–623. https://doi.org/10.1002/gcc.20669</p>

<p>Lahti, L., Myllykangas, S., Knuutila, S., &amp; Kaski, S. (2009). Dependency detection with similarity constraints. In J. C. Adali T Chanussot J &amp; L. J (Eds.), <i>Proc. MLSP’09 IEEE International Workshop on Machine Learning for Signal Processing XIX</i> (pp. 198–203). Piscataway, NJ: software. Retrieved from http://arxiv.org/abs/1101.5919</p>
